README.md

```diff
@@ -191,7 +181,7 @@ final drawing = SpinnerDrawing(
   left: 300,
   top: 200,
   scale: 2.5,
-  color: Colors.blue,
+  colorHex: Colors.blue.value,
```

Comment:
Can you add a word to explain why we use double value and not Flutter colors in the readme ?